client/utils/planning.tsx

```diff
@@ -1789,7 +1789,7 @@ function addToWorkflowCommon<T extends IPlanningCoverageItem | ICoverageSchedule
     next.workflow_status = COVERAGES.WORKFLOW_STATE.ACTIVE;
 
     // Scheduled update does not have `add_coverage_to_workflow`
-    if ('original_coverage_id' in (item as IPlanningCoverageItem)) {
+    if (!('scheduled_update_id' in item)) {
```

Comment:
use [nameof](https://github.com/superdesk/sourcefabric-common/commit/b2e2e166676c8f4a0e394c68f9727372a2be79c6) to ensure type safety of keys. It was also in superdeskApi, but I found there are sometimes issues with accessing superdeskApi in tests thus I'm moving generic helpers to @sourcefabric/common